mungegithub/mungers/approval-handler.go

```diff
@@ -202,13 +202,27 @@ func (h ApprovalHandler) unwrapAliases(ownerSet sets.String) sets.String {
 	return aliases.Expand(ownerSet)
 }
 
+func getRandomizedKeys(approverOwnersfiles map[string]sets.String) []string {
```

Comment:
I'd really really like to see a test function before we get this merged.

Also, i see this doesn't depend on a seed, but on the number of files. Does it mean that the same set of files will always lead to the same approver? Is that fair?